.gitignore

```diff
@@ -12,3 +12,5 @@ node_modules/
 
 # OS Specific
 .DS_Store
+.vscode/launch.json
+dnd5e.code-workspace
```

Comment:
These can be removed, as a PR to add the `.vscode` folder to .gitignore is [currently open](https://github.com/foundryvtt/dnd5e/pull/1923).